param/parameterized.py

```diff
@@ -2677,13 +2702,15 @@ def __str__(self):
         return "<%s %s>" % (self.__class__.__name__,self.name)
 
 
+    @recursive_repr()
```

Comment:
Actually, I don't think this one's needed, because it's not recursive; it just calls self.pprint, and self.pprint can handle the recursion check.
```suggestion
```